website/docs/r/express_route_port_authorization.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_express_route_port_authorization"
+description: |-
+  Manages an ExpressRoute Port Authorization.
+---
+
+# azurerm_express_route_port_authorization
+
+Manages an ExpressRoute Port Authorization.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "exprtTest"
+  location = "West Europe"
+}
+
+resource "azurerm_express_route_port" "example" {
+  name                = "port1"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  peering_location    = "Airtel-Chennai-CLS"
+  bandwidth_in_gbps   = 10
+  encapsulation       = "Dot1Q"
+}
+
+resource "azurerm_express_route_port_authorization" "example" {
+  name                    = "exampleERCAuth"
+  express_route_port_name = azurerm_express_route_port.example.name
+  resource_group_name     = azurerm_resource_group.example.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the ExpressRoute Port. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the ExpressRoute Port. Changing this forces a new resource to be created.
+    
+
+* `express_route_port_name` - (Required) The name of the Express Route Port in which to create the Authorization. Changing this forces a new resource to be created.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the ExpressRoute Port Authorization.
+
+* `authorization_key` - The Authorization Key.
+
+* `authorization_use_status` - The authorization use status.
```

Comment:
could you expand on what this property does?

and does it make sense to be exposed in terraform?

Reply:
This is the authorization status indicate if this authorization is in `use` or `available`.  This resource is similar to [azurerm_express_route_circuit_authorization](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/express_route_circuit_authorization)

Reply:
👍  thanks